<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProtoIt help creator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ProtoItHelpCreator.ico</normaloff>ProtoItHelpCreator.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ProtoIt Device-bestand (*.pid)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFile">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Maak 'model.html' infobestand</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstScript">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edtScript">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>340</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edtInfo">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>351</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Modelbeschrijving</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Script-items</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>305</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Beschrijving van: &lt;PORT&gt;, &lt;ADDRESS&gt;, &lt;CALIBRATE&gt;,
                          &lt;SIGNALIN&gt; en &lt;SIGNALOUT&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnHelp1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Voorbeeld</string>
    </property>
   </widget>
   <widget class="QWebView" name="wvwViewer">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>541</width>
      <height>481</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
